src/components/cadastrar-doador.tsx

```diff
@@ -2,6 +2,7 @@
 
 import React from "react";
 import axios from "axios";
+import Image from "next/image";
```

Comment:
Muito bem!
Sempre que for inserir uma imagem em alguma página (no framework Next), utilize o componente <Image/> do Next.

Este componente otimiza a imagem de acordo com a altura e largura passada como prop, de maneira que a imagem não tenha mais 'peso' (resolução) do que o necessário para atender as dimensões solicitadas.